e2e/runner/setup_solana.go

```diff
@@ -85,6 +85,30 @@ func (r *E2ERunner) SetupSolana(deployerPrivateKey string) {
 	require.NoError(r, err)
 	r.Logger.Info("initial PDA balance: %d lamports", balance.Value)
 
+	// initialize rent payer
+	var instRentPayer solana.GenericInstruction
+	rentPayerPdaComputed := r.ComputeRentPayerPdaAddress()
```

Comment:
nit: Same as before, since the E2ERunner is multichain it would make sense to group helper functions by name.